src/test/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandlerTest.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.barclay.help;
+
+import org.broadinstitute.barclay.help.testinputs.TestArgumentContainer;
+import org.broadinstitute.barclay.help.testinputs.TestExtraDocs;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+/**
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public class DefaultDocWorkUnitHandlerTest {
+
+    private final static DefaultDocWorkUnitHandler DEFAULT_DOC_WORK_UNIT_HANDLER =
+            new DefaultDocWorkUnitHandler(new HelpDoclet());
+
+    @DataProvider
+    public Object[][] workUnitForDocStrings() throws Exception {
+        return new Object[][] {
+                {DocGenMocking.createDocWorkUnit(DEFAULT_DOC_WORK_UNIT_HANDLER, TestArgumentContainer.class),
+                        "", "Argument container class for testing documentation generation.",
+                        "Test feature group name", "Test program group used for testing"},
```

Comment:
It took me a little hunting to figure where these strings came from. They should be factored out into symbolic constants (probably defined in the test input class) that can be used both here and in the test input class. Then its easy to see why they're the expected values, and they'll be prefixed with, say "TestArgumentContainer." so they'll be easily navigable in the IDE.

Reply:
Done.